config/schema.yaml

```diff
@@ -694,19 +694,20 @@ schema:
         type: string
         doc: A string value composed only of letters, numbers, and underscores.
 
-  dataunit_joins:
-    ExposureRangeJoin:
-      doc: >
-        A calculated join between Datasets identified with an Exposure
-        (typically raw science frames) and Datasets identified with ExposureRange
-        (typically master calibrations).
-      lhs:
-      - Exposure
-      rhs:
-      - ExposureRange
-      sql:
-        (lhs.camera = rhs.camera) AND
-        (lhs.exposure BETWEEN rhs.valid_first AND rhs.valid_last)
+  dataUnitJoins:
+    # TODO: Write a proper table/view definition for this
+    # ExposureRangeJoin:
+    #   doc: >
+    #     A calculated join between Datasets identified with an Exposure
+    #     (typically raw science frames) and Datasets identified with ExposureRange
+    #     (typically master calibrations).
+    #   lhs:
+    #   - Exposure
+    #   rhs:
+    #   - ExposureRange
+    #   sql:
+    #     (lhs.camera = rhs.camera) AND
+    #     (lhs.exposure BETWEEN rhs.valid_first AND rhs.valid_last)
```

Comment:
Instead of commenting this out, let's just leave this as it was, and ignore it when building the schema for now.  I don't think we will ever want to make it a view; I think we'll instead want to find some way to inject this join expression into preflight queries.

I think since this "sql" entry is not inside a "tables" entry, this shouldn't be parsed by the DataUnitJoin code; if it is, we should fix that instead.

Reply:
Yes, it is skipped.

Reply:
I just commented it out such that the skipping is more explicit.